onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -1494,4 +1494,59 @@ TEST(CApiTest, ConfigureCudaArenaAndDemonstrateMemoryArenaShrinkage) {
   // (Memory arena for the CPU should not have been disabled)
   //  run_option.AddConfigEntry(kOrtRunOptionsConfigEnableMemoryArenaShrinkage, "cpu:0;gpu:0");
 }
-#endif
\ No newline at end of file
+#endif
+
+//#ifdef USE_TENSORRT
```

Comment:
is the ifdef needed? if not, can we remove the comment?

Reply:
It's needed. uncommented it.